# Track Awesome Biological Visualizations Updates Weekly

A list of web-based interactive biological data visualizations.

[üè† Home](/README.md) ¬∑ [üîç Search](https://test.trackawesomelist.com/search/) ¬∑ [üî• Feed](https://test.trackawesomelist.com/keller-mark/awesome-biological-visualizations/week/rss.xml) ¬∑ [üìÆ Subscribe](https://trackawesomelist.us17.list-manage.com/subscribe?u=d2f0117aa829c83a63ec63c2f&id=36a103854c) ¬∑ [üò∫ keller-mark/awesome-biological-visualizations](https://github.com/keller-mark/awesome-biological-visualizations/blob/main/README.md) ¬∑ ‚≠ê 97 ¬∑ üè∑Ô∏è Miscellaneous

[ [Daily](/content/keller-mark/awesome-biological-visualizations/README.md) / Weekly / [Overview](/content/keller-mark/awesome-biological-visualizations/readme/README.md) ]



## [Mar 29 - Apr 04, 2021](/content/2021/13/README.md)

### Genomics

*   [Gosling](https://gosling.js.org) - A grammar-based toolkit for scalable and interactive genomics data visualization.

## [Feb 22 - Feb 28, 2021](/content/2021/8/README.md)

### 3D Genomics

*   [HiPiler](http://hipiler.higlass.io/) - A tool for visually exploring and aggregating reoccurring patterns (like loops, TADs, etc.) in Hi-C maps.

### Epigenomics

*   [Peax (‚≠ê63)](https://github.com/Novartis/peax) - Interactive visual pattern exploration in epigenomic data using unsupervised deep representation learning.

### Mass Spectrometry / Gene expression matrices

*   [Cytosplore](https://www.cytosplore.org/) - Cytosplore is an interactive visual analysis system for the analysis of mass cytometry data.

## [Jan 04 - Jan 10, 2021](/content/2020/53/README.md)

### Genomics

*   [Coessentiality](http://coessentiality.net) - A genome-wide almanac of co-essential modules assigns function to uncharacterized genes.

## [Dec 21 - Dec 27, 2020](/content/2020/51/README.md)

### (Non-spatial) Transcriptomics / Gene expression matrices

*   [EBI Single Cell Expression Atlas](https://www.ebi.ac.uk/gxa/sc/home) - Single cell gene expression across species.
*   [MORPHEUS](https://software.broadinstitute.org/morpheus/) - Versatile matrix visualization and analysis software.
*   [Clustergrammer](https://maayanlab.cloud/clustergrammer/) - Visualize high-dimensional data as interactive hierarchically clustered heatmaps.

## [Oct 05 - Oct 11, 2020](/content/2020/40/README.md)

### (Non-spatial) Transcriptomics

*   [TISCH](http://tisch.comp-genomics.org/home/) - Tumor Immune Single-cell Hub (TISCH) is a scRNA-seq database focusing on tumor microenvironment.

## [Sep 07 - Sep 13, 2020](/content/2020/36/README.md)

### Cancer

*   [cBioPortal for Cancer Genomics](http://www.cbioportal.org/) - Visualization, analysis and download of large-scale cancer genomics data sets.
*   [Oncoscape](https://oncoscape.sttrcancer.org/) - An integrated suite of analysis tools for users to explore hypotheses related to molecular and clinical data.

### Genomics

*   [PLACNETw](https://castillo.dicom.unican.es/ex1/?unique=58f731fec480a) - A graph-based tool for reconstruction of plasmids from next generation sequence pair-end datasets.
*   [PhyD3](https://phyd3.bits.vib.be/view.php?id=91162629d258a876ee994e9233b2ad87\&f=xml) - A phylogenetic tree viewer.
*   [Archaeopteryx](http://www.phyloxml.org/archaeopteryx-js/bcl2_js.html) - Visualization, analysis, and editing of phylogenetic trees.

### 3D Genomics

*   [TADkit](http://sgt.cnag.cat/3dg/tadkit/demo.h/index.html#!/project/dataset?conf=assets%2Fexamples%2Fconf.json) - 3D genome browser and TADbit front-end.

### Proteomics

*   [COSMIC-3D](https://cancer.sanger.ac.uk/cosmic3d/protein/EGFR) - A platform for understanding cancer mutations in the context of 3D protein structure.

### Spatial transcriptomics

*   [Vitessce](http://vitessce.io) - Visual integration tool for exploration of spatial single-cell experiment data.
*   [Brain mRNA Isoform Atlas](https://isoformatlas.com/) - Visualize cell type signatures of alternative splicing in postnatal development.
*   [AlzMap](https://alzmap.org/) - Spatial transcriptomics in Alzheimer's disease.
*   [Loom Viewer](http://loom.linnarssonlab.org/dataset/cells/osmFISH/osmFISH_SScortex_mouse_all_cells.loom/NrBEoXQGhYDYoAYLRARigZlRgTDpAtAgFhRnSyg0Wrt1rUVuAHYlCYAOD5AzOo3qNmMAJy8ctaeWAYS0jvNoNqotpO6TSioZgmqmLCTJhNt0ebPCpgAWkQA6RJjgkxYxLkwlMrAKxIjr5wbv7\~iKxcaLj\~0AIkULhK\~DAscpwZxGZJ_LIgigB2AK4ANqVKHIbquqi10LVKsgIl5RxyybFQPIGYGAi4yey4PGjDArjsaAgYaBOJaDy4A3i0C4G4EgbUEmiJmIwYmMnm2E0QQA) - Tool for sharing, browsing and visualizing single-cell data stored in the Loom file format.
*   [TissUUmaps](https://tissuumaps.research.it.uu.se/) - Interactive visualization of large-scale spatial gene expression and tissue morphology data.
*   [Giotto Viewer](http://spatialgiotto.rc.fas.harvard.edu/giotto.viewer.html) - A web-based visualization package for spatial transcriptomic data.

### (Non-spatial) Transcriptomics

*   [cellxgene](https://www.kidneycellatlas.org/mature-kidney-immune) - An interactive explorer for single-cell transcriptomics data.
*   [UCSC Cell Browser](http://cells.ucsc.edu/?ds=cortex-dev) - An interactive viewer for single-cell expression.
*   [ASAP](https://asap.epfl.ch/) - A collaborative portal to analyze single-cell transcriptomics data.
*   [Mouse Brain Atlas](http://mousebrain.org/) - Atlas of cell types from the Linnarsson Lab.
*   [Broad Institute Single Cell Portal](https://singlecell.broadinstitute.org/single_cell) - Reducing barriers and accelerating single-cell research.
*   [Allen Brain Map - Transcriptomics Explorer](http://celltypes.brain-map.org/rnaseq/human_m1_10x) - Atlas of human and mouse transcriptomic cell types.
*   [Single Cell VR](https://singlecellvr.herokuapp.com/) - Interactive visualization of single-cell data in virtual reality.
*   [SPRING](https://kleintools.hms.harvard.edu/tools/springViewer_1_6_dev.html?datasets/mouse_HPCs/basal_bone_marrow/full) - A tool for uncovering high-dimensional structure in single-cell gene expression data.
*   [START](https://kcvi.shinyapps.io/START/) -  Shiny Transcriptome Analysis Resource Tool allows users to visualize RNA-seq data starting with count data.
*   [iSEE](https://marionilab.cruk.cam.ac.uk/iSEE_allen/) - The Interactive SummarizedExperiment Explorer.

### Microscopy / Gene expression matrices

*   [IONpath MIBItracker](https://www.ionpath.com/mibitracker/) - Slide and image data.
*   [Avivator](http://avivator.gehlenborglab.org/) - A lightweight "batteries-included" WebGL viewer for remote imaging data in Bio-Formats Zarr or OME-TIFF format.

### Connectomics / Gene expression matrices

*   [Allen Mouse Brain Connectivity Atlas](http://connectivity.brain-map.org/) - A high-resolution map of neural connections in the mouse brain.
*   [FlyWire](https://flywire.ai/) - A human-AI collaboration to map the fly connectome.

### Metabolomics / Gene expression matrices

*   [Metaspace](https://metaspace2020.eu/) - A platform for metabolite annotation of imaging mass spectrometry data.

### Medicine / Gene expression matrices

*   [Snake Oil Supplements](http://informationisbeautiful.net/visualizations/snake-oil-scientific-evidence-for-nutritional-supplements-vizsweet/) - Scientific evidence for popular health supplements.
*   [SieveSifter](http://sieve.fredhutch.org/viz/index.html?study=VTN503\&protein=gag\&reference=MRK_B_Ad5) - Tool for visualizing the sieve analyses of HIV-1 vaccine efficacy trials.

## [Mar 23 - Mar 29, 2020](/content/2020/12/README.md)

### Cancer

*   [Signal](https://signal.mutationalsignatures.com/) - Mutational signatures extracted from cancer samples and isogenic cell-based experiments.

### (Non-spatial) Transcriptomics

*   [SCope](http://scope.aertslab.org/) - A visualization tool for large-scale scRNA-seq datasets.

### Connectomics / Gene expression matrices

*   [neuroglancer](https://hemibrain-dot-neuroglancer-demo.appspot.com/#!gs://neuroglancer-janelia-flyem-hemibrain/v1.0/neuroglancer_demo_states/kc_apl_mpn1.json) - Tool for visualization of highly detailed maps of neuronal connectivity in the fly brain.

## [Oct 07 - Oct 13, 2019](/content/2019/40/README.md)

### Cancer

*   [ExploSig](http://explosig.lrgr.io) - Interactive visualization of mutational signatures and clinical data.

## [Jun 25 - Jul 01, 2018](/content/2018/26/README.md)

### Genomics

*   [GeneMANIA](http://genemania.org/) - Helps predict the function of genes and gene sets.
*   [Mutaframe](http://deogen2.mutaframe.com/) - Visualize variants of the human genome.

### Epigenomics

*   [GIVe](https://mcf7.givengine.org/) - Epigenome and genome interaction.

## [May 21 - May 27, 2018](/content/2018/21/README.md)

### Cancer

*   [COSMIC Cancer Browser](https://cancer.sanger.ac.uk/cosmic/browse/tissue) - Browse COSMIC data by tissue type and histology.
*   [UCSC Xena](https://xenabrowser.net/) - Explore functional genomic data sets for correlations between genomic and/or phenotypic variables.
*   [iCoMut Beta for FireBrowse](http://firebrowse.org/iCoMut/) - Interactive exploration of mutation rates, mutation signatures, clinical data, gene mutations.
*   [MAGI](http://magi.brown.edu) - Visualization and collaborative annotation of genomic aberrations.
*   [Race, Gender, and Age in TCGA](https://www.enpicom.com/visual-lab/tcga-visual-exploration-gender-race-age/) - An interactive visualization of patient gender, race, and age in The Cancer Genome Atlas (TCGA) dataset.
*   [OASISPRO](http://tinyurl.com/oasispro) - Omics analysis tool for clinical phenotype prediction.

### Genomics

*   [UCSC Genome Browser](https://genome.ucsc.edu/) - Interactively visualize genomic data.
*   [Ensembl](http://www.ensembl.org) - A genome browser for vertebrate genomes that supports research in comparative genomics, evolution, sequence variation and transcriptional regulation.
*   [IMP: Integrative Multi-species Prediction](http://imp.princeton.edu/) - Analyze your experimental results in the functional context of gene-gene networks from multiple organisms.

### 3D Genomics

*   [HiGlass](http://higlass.io/) - A tool for exploring genomic contact matrices and tracks.
*   [Delta](http://delta.big.ac.cn/) - An integrative analysis and visualization platform for 3D genome.
*   [Juicebox](http://aidenlab.org/juicebox/) - Visualization for Hi-C data.

### Metagenomics

*   [Metaviz](http://metaviz.cbcb.umd.edu/) - Interactive statistical and visual analysis of metagenomic data.
*   [Malaria Cell Atlas](https://www.sanger.ac.uk/science/tools/mca/mca/) - Explore how genes are expressed across more than 500 individual Plasmodium parasites.

### Epigenomics

*   [Epiviz](http://epiviz.cbcb.umd.edu/4/) - Interactive visual analytics for functional genomics data.
*   [WashU Epigenome Browser](http://epigenomegateway.wustl.edu/) - Displays many sequencing-based data sets across all or part of the genome, on specific gene sets or pathways, and in the context of their metadata.
*   [Clustergrammer CCLE Explorer](https://maayanlab.github.io/CCLE_Clustergrammer/) - Browse relative mRNA expression data for cancer cell lines from the Cancer Cell Line Encyclopedia (CCLE) by tissue of origin using Clustergrammer.

### Proteomics

*   [EzMol](http://www.sbg.bio.ic.ac.uk/\~ezmol/) - Visualisation and image production of protein and nucleic acid structures.
*   [LiteMol](https://webchemdev.ncbr.muni.cz/LiteMol/) - Visualization of large-scale
    macromolecular structure data.

### (Non-spatial) Transcriptomics

*   [L1000FWD](http://amp.pharm.mssm.edu/L1000FWD/) - Large-scale visualization of drug-induced transcriptomic signatures.

### Populations / Gene expression matrices

*   [Nextstrain](https://nextstrain.org/) - Real-time tracking of virus evolution.
*   [Institute for Health Metrics and Evaluation](http://www.healthdata.org/results/data-visualizations) - Visualizations of disease worldwide.
*   [Lifespans](http://www.cotrino.com/lifespan/) - Visualization of the lifespans of multiple species.
*   [Life Expectancy](http://projects.flowingdata.com/life-expectancy/) - Life expectancies around the world, 1960 to 2009.